doc/importexport.rst

```diff
@@ -0,0 +1,263 @@
+* *height* - Document height of the resulting image.
+* *marginLeft* - Inset margin from the left side of the document.
+* *marginTop* - Inset margin from the top side of the document.
+* *projectionDir* - Direction the camera will view the shape from.
+* *showAxes* - Whether or not to show the axes indicator, which will only be visible when the projectionDir is also at the default.
+* *strokeWidth* - Width of the line that visible edges are drawn with.
+* *strokeColor* - Color of the line that visible edges are drawn with.
+* *hiddenColor* - Color of the line that hidden edges are drawn with.
+* *showHidden* - Whether or not to show hidden lines.
+
+The options are passed to the exporter in a dictionary, and can be left out to force the SVG to be created with default options. 
+Below are examples with and without options set.
+
+Without options:
+
+.. code-block:: python
+
+    import cadquery as cq
+    from cadquery import exporters
+
+    result = cq.Workplane().box(10, 10, 10)
+
+    exporters.export(result, '/path/to/file/box.svg')
+
+Results in:
+
+..  image:: _static/importexport/box_default_options.svg
+
+Note that the exporters API figured out the format type from the file extension. The format 
+type can be set explicitly by using :py:class:`exporters.ExportTypes`.
+
+The following is an example of using options to alter the resulting SVG output by passing in the ``opt`` parameter.
+
+.. code-block:: python
+
+    import cadquery as cq
+    from cadquery import exporters
+
+    result = cq.Workplane().box(10, 10, 10)
+
+    exporters.export(
+                result,
+                '/path/to/file/box_custom_options.svg',
+                opt={
+                    "width": 300,
+                    "height": 300,
+                    "marginLeft": 10,
+                    "marginTop": 10,
+                    "showAxes": False,
+                    "projectionDir": (0.5, 0.5, 0.5),
+                    "strokeWidth": 0.25,
+                    "strokeColor": (255, 0, 0),
+                    "hiddenColor": (0, 0, 255),
+                    "showHidden": True,
+                },
+            )
+
+Which results in the following image:
+
+..  image:: _static/importexport/box_custom_options.svg
+
+Exporting STL
+##############
+
+The STL exporter is capable of adjusting the quality of the resulting mesh, and accepts the following parameters.
+
+* ``fileName`` - The path and file name to write the STL output to.
+* ``tolerance`` - A linear deflection setting which limits the distance between a curve and its tessellation. Setting this value too low will result in large meshes that can consume computing resources. Setting the value too high can result in meshes with a level of detail that is too low. Default is 0.1, which is good starting point for a range of cases.
+* ``angularTolerance`` - Angular deflection setting which limits the angle between subsequent segments in a polyline. Default is 0.1.
```

Comment:
LGTM. I would kind of rather this information was on the `exporters.export` docstring and use automethod to stick it in here, but that is just minor.

Reply:
@marcus7070 My rst-fu is weak so I didn't know that parameters from the docstring could be included. I'll add docstrings and do the same thing you did with importDXF.

Reply:
The best I could do was add a docstring to `Shape.exportStl()` and include that. AMF and VRML are tessellated inside the `exporters()` method, so there's no place to add a docstring for them. Those parameter listings will just have to be maintained separately.  At least `exportStl()` got a docstring out of it.